docs/config-definition.md

```diff
@@ -1,9 +1,9 @@
 # Ark Config definition
 
-* [Overview][10]
```

Comment:
We don't >< I think the numbers might make it confusing (because people assume that it needs to be in order when it doesn't, @madddi could just append the new link to the end of the numbered list). Essentially the stuff at the bottom are K/V pairs (shorthands and their links), which are most useful when you are reusing the same link multiple times throughout a doc.

This is probably fine as is but let me know if you think we should add some sort of note about this to **Contributing** or something @ncdc